tests/test_jwt.py

```diff
@@ -59,7 +59,7 @@ def test_sign_and_verify(self):
     self._check_sign_and_verify('privatekey.%s' % self.format)
 
   def test_sign_and_verify_from_converted_pkcs12(self):
-    """Tests that following instructions to convert from PKCS12 to PEM works."""
```

Comment:
Docstrings in tnittest-based testcase methods are a bad idea because (historically, anyway) testrunners tend to display the docstring, rather than the name of the testcase, in output, making it hard to diagnose failures.